operator/pkg/psp/emm_test.go

```diff
@@ -0,0 +1,322 @@
+// Copyright DataStax, Inc.
+// Please see the included license file for details.
+
+package psp
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+
+	"github.com/go-logr/logr"
+	logrtesting "github.com/go-logr/logr/testing"
+	
+
+	"github.com/datastax/cass-operator/operator/internal/result"
+)
+
+func IsRequeue(t *testing.T, r result.ReconcileResult, msg string) {
+	isRequeue := false
+	if r.Completed() == true {
+		v, err := r.Output()
+		if err == nil {
+			isRequeue = v.Requeue
+		}
+	}
+	require.True(t, isRequeue, msg)
+}
+
+func IsContinue(t *testing.T, r result.ReconcileResult, msg string) {
+	require.False(t, r.Completed(), msg)
+}
+
+type MockEMMService struct {
+	mock.Mock
+}
+
+func (m *MockEMMService) getRacksWithNotReadyPodsBootstrapped() []string {
+	args := m.Called()
+	return args.Get(0).([]string)
+}
+
+func (m *MockEMMService) getNodeNameSetForRack(rackName string) map[string]bool {
```

Comment:
many uses of `map[string]bool` - can we do something like
```
type map[string]bool StringSet
```
for more readable code?

Reply:
(just left a comment on the first one that comes up on GH, nothing special about this one)